migration/migration.c

```diff
@@ -2117,7 +2158,7 @@ bool migrate_cuju_enabled(void)
     return s->enabled_capabilities[MIGRATION_CAPABILITY_CUJU_FT];
 }
 
-static void migrate_fd_get_notify(void *opaque)
+/*static void migrate_fd_get_notify(void *opaque)
```

Comment:
Please do not comment out original migrate_fd_get_notify and create a new one
Direct modify this function

Reply:
add a new function which calls "cuju_migrate_fd_get_notify" and comment out the original "migrate_fd_get_notify" function

Reply:
original migrate_fd_get_notify？

Reply:
because original migrate_fd_get_notify not used if I don't comment out it would get error:
not used but declared.